src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -193,9 +276,15 @@ protected Object doWork() {
 
         return null;
     }
+*/
+    // local files on the client's disk -> PCollections
+    private void ingestInputs(final Pipeline pipeline) throws IOException {
+
+        referencePath = BRAC.referenceArguments.getReferenceFileName();
+        // TODO: support more than one input.
+        beforePath = BRAC.readArguments.readFilesNames.get(0);
```

Comment:
Also, if you don't support more than one read input, you should throw if you get more than one.